pkg/common/apis/header.go

```diff
@@ -77,6 +77,10 @@ func GetUserID(ctx context.Context) string {
 	return GetHeader(ctx, "user-id")
 }
 
+func GetMSPInfo(ctx context.Context, key string) string {
+	return GetHeader(ctx, key)
```

Comment:
you should use a more common method name, such as `GetValueByKey` or `GetValue`.

Reply:
You are right, I will create a new pr to fix this method naming.